cppConsole/Dockerfile

```diff
@@ -12,7 +12,7 @@ ARG APP_ROOT=
 
 
 # BUILD ------------------------------------------------------------------------
-FROM commontorizon/debian-cross-toolchain-${IMAGE_ARCH}:${CROSS_SDK_BASE_TAG} AS build
+FROM torizon/debian-cross-toolchain-${IMAGE_ARCH}:${CROSS_SDK_BASE_TAG} AS build
```

Comment:
Here we should use `commontorizon` as an exception, because the container for arm is `debian-cross-toolchain-armhf` instead of `debian-cross-toolchain-arm`. Also, the tags are different for `debian-cross-toolchain-armhf` and `debian-cross-toolchain-arm64`. `3.2.0` for example is not available for `debian-cross-toolchain-armhf, `3.3.0` and beyond is not available for `debian-cross-toolchain-arm64`, one option for this problem would be to use just the major `3` though.

Reply:
This should be handled by 86d22b3

Reply:
Some adjustment on build steps need to be also added: 4a5b4dd

Reply:
Sounds good this adjustments